digiroad2-oracle/src/main/resources/db/migration/V236_create_new_roawday_asset.sql

```diff
@@ -0,0 +1,441 @@
+-- NEW ASSET TYPE
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (450, 'K1 keskiviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (460, 'K2 ajokaistaviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (470, 'K3 Sulkuviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (480, 'K4 Varoitusviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (490, 'K5 Sulkualue', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (500, 'K6 Reunaviiva', 'linear', 'db_migration_v236');
+
+-- LOCALIZED_STRING VALUE
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K1 keskiviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K2 ajokaistaviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K3 Sulkuviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K4 Varoitusviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K5 Sulkualue', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K6 Reunaviiva', 'db_migration_v236', SYSDATE);
```

Comment:
The LOCALIZED_STRING values should be the text to be used as name of the field in the form in the UI, in those cases, you are adding values as a name of fields.

For example, you need to check the fields to have at the form, lets use the "Relative location / Suhteellinen sijainti" example.

For this case, you need to verify if already exist some LOCALIZED_STRING (field label) like "Suhteellinen sijainti", if yes, you don't need to add it, but if not you need to do something like:

```SQL
-- LOCALIZED_STRING VALUE
INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
	VALUES( primary_key_seq.nextval, 'Suhteellinen sijainti', 'db_migration_v236', SYSDATE );